### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by pellared**
## Why

> Why this pull request was created? Explain the problem that this pull request is trying to solve.

Make it easier to create a meaningful pull request description.

## What

> What does this pull request contain? Explain what is this pull request changing.

Add pull request template.

### CODE REVIEW

**Review by agabrys — CHANGES_REQUESTED** [review]
Great template - short, simple and provides a lot of information 👍

I added one proposal how to improve it a little bit (hide the sections' description).

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Why
+
+> Why this pull request was created? Explain the problem that this pull request is trying to solve.
+
+## What
+
+> What does this pull request contain? Explain what is this pull request changing.
```
[the remark this answers is not in the document]
Comment (agabrys) [reply .github/pull_request_template.md:7]: I checked you goyek repo and now I understand you would like to have these comments always visible:

goyek-pr

I think it has sense, so this one could be ignored.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+## Checklist
+
+- [ ] `README.md` is updated.
```
[the remark this answers is not in the document]
Comment (pellared) [reply .github/pull_request_template.md]: Please check: https://github.com/agabrys/metadata-api-plugin/pull/1/commits/eb0b33e845e475eddc81bcef9611e6fa3606a638

**Review by agabrys — APPROVED** [review]
Great, thank you 👍

**Pull request merged**